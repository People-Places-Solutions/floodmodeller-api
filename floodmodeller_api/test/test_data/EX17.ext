[QTBDY__S1.QT]
X=200500
Y=52284.375

[FEHBDY__T1]
X=200900
Y=52300

[QTBDY__M1]
X=200500
Y=52268.75

[RIVER_MUSKINGUM_M1]
X=200500
Y=52253.125

[RIVER_MUSKINGUM_M2]
X=200500
Y=52237.5

[JUNCTION_OPEN_M2]
X=200500
Y=52221.875

[RIVER_SECTION_S1]
X=200500
Y=52206.25

[RIVER_SECTION_S2]
X=200500
Y=52190.625

[RIVER_SECTION_S3]
X=200500
Y=52175

[RIVER_SECTION_S4Up]
X=200500
Y=52159.375

[JUNCTION_OPEN_S4Up]
X=200500
Y=52143.75

[WEIR__S4_W]
X=200500
Y=52128.125

[RIVER_SECTION_S4Dn]
X=200500
Y=52112.5

[RIVER_SECTION_S5]
X=200500
Y=52096.875

[RIVER_SECTION_S6]
X=200500
Y=52081.25

[RIVER_SECTION_S7]
X=200500
Y=52065.625

[RIVER_SECTION_S8]
X=200500
Y=52050

[QHBDY__S8]
X=200500
Y=52034.375

[QTBDY__Lateral_U]
X=200500
Y=52018.75

[LATERAL__Lateral_U]
X=200500
Y=52003.125

[QTBDY__Lateral_D]
X=200500
Y=51987.5

[LATERAL__Lateral_D]
X=200500
Y=51971.875

[LATERAL__Lateral_Trib]
X=200500
Y=51956.25

[QTBDY__Lateral_Trib]
X=200500
Y=51940.625

[RIVER_SECTION_T1]
X=200500
Y=51925

[RIVER_SECTION_T2u]
X=200500
Y=51909.375

[CULVERT_INLET_T2u]
X=200500
Y=51893.75

[CONDUIT_RECTANGULAR_T2]
X=200500
Y=51878.125

[CONDUIT_RECTANGULAR_T2d]
X=200500
Y=51862.5

[CULVERT_OUTLET_T2d]
X=200500
Y=51846.875

[RIVER_SECTION_T3]
X=200500
Y=51831.25

[RIVER_SECTION_T4]
X=200500
Y=51815.625

[Connections]
ConnectionCount=37
1=QTBDY__S1.QT,JUNCTION_OPEN_M2
2=FEHBDY__T1,RIVER_SECTION_T1
3=QTBDY__M1,RIVER_MUSKINGUM_M1
4=RIVER_MUSKINGUM_M1,RIVER_MUSKINGUM_M2
5=RIVER_MUSKINGUM_M2,JUNCTION_OPEN_M2
6=JUNCTION_OPEN_M2,RIVER_SECTION_S1
7=RIVER_SECTION_S1,RIVER_SECTION_S2
8=RIVER_SECTION_S2,RIVER_SECTION_S3
9=RIVER_SECTION_S3,RIVER_SECTION_S4Up
10=RIVER_SECTION_S4Up,JUNCTION_OPEN_S4Up
11=JUNCTION_OPEN_S4Up,RIVER_SECTION_T4
12=JUNCTION_OPEN_S4Up,WEIR__S4_W
13=WEIR__S4_W,RIVER_SECTION_S4Dn
14=RIVER_SECTION_S4Dn,RIVER_SECTION_S5
15=RIVER_SECTION_S5,RIVER_SECTION_S6
16=RIVER_SECTION_S6,RIVER_SECTION_S7
17=RIVER_SECTION_S7,RIVER_SECTION_S8
18=RIVER_SECTION_S8,QHBDY__S8
19=QTBDY__Lateral_U,LATERAL__Lateral_U
20=LATERAL__Lateral_U,RIVER_SECTION_S1
21=LATERAL__Lateral_U,RIVER_SECTION_S2
22=QTBDY__Lateral_D,LATERAL__Lateral_D
23=LATERAL__Lateral_D,RIVER_SECTION_S4Dn
24=LATERAL__Lateral_D,RIVER_SECTION_S5
25=LATERAL__Lateral_D,RIVER_SECTION_S6
26=LATERAL__Lateral_D,RIVER_SECTION_S7
27=LATERAL__Lateral_Trib,QTBDY__Lateral_Trib
28=LATERAL__Lateral_Trib,RIVER_SECTION_T1
29=LATERAL__Lateral_Trib,RIVER_SECTION_T3
30=LATERAL__Lateral_Trib,RIVER_SECTION_S3
31=RIVER_SECTION_T1,RIVER_SECTION_T2u
32=RIVER_SECTION_T2u,CULVERT_INLET_T2u
33=CULVERT_INLET_T2u,CONDUIT_RECTANGULAR_T2
34=CONDUIT_RECTANGULAR_T2,CONDUIT_RECTANGULAR_T2d
35=CONDUIT_RECTANGULAR_T2d,CULVERT_OUTLET_T2d
36=CULVERT_OUTLET_T2d,RIVER_SECTION_T3
37=RIVER_SECTION_T3,RIVER_SECTION_T4

[VISTEXT]
VISTEXTCOUNT=0

[Font]
Font name=Tahoma
Charset=1
Height=-11
Bold=0
Italic=0
Underline=0
Strikeout=0

[Text]
Layout=1

[Display]
MiniNodes=0
MiniStructureNodes=0
NodeLabels=0
Connections=1

[General]
UnitMove=0
ExportConnections=0
GeneralConnectionsColour=0
LateralConnectionsColour=16711680
SpillConnectionsColour=8421504

[Grid]
Show=0
X=25
Y=25

[RWC]
Top=386153.846153846
Left=-364181.14225556
Right=288269.838136597
Bottom=13846.1538461538

[PC]
Top=0
Left=0
Right=0
Bottom=0

