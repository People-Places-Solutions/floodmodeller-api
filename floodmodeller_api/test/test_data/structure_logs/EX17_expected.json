{
    "(T2,CONDUIT)": {
        "name": "T2",
        "type": "CONDUIT",
        "subtype": "RECTANGULAR",
        "comment": "",
        "conduit_data": {
            "length": 30.0,
            "inlet": 0.6,
            "total_length": 30.0
        },
        "dimensions": {
            "invert": 47.5,
            "height": 1.5,
            "width": 2.5
        },
        "friction": {
            "friction_eq": "MANNING",
            "friction_set": [
                0.01,
                0.015,
                0.02
            ],
            "all_friction": [
                0.01,
                0.02,
                0.015
            ]
        }
    },
    "(T2d,CONDUIT)": {
        "name": "T2d",
        "type": "CONDUIT",
        "subtype": "RECTANGULAR",
        "comment": "",
        "conduit_data": {
            "length": 0.0,
            "outlet": 1.0
        },
        "dimensions": {
            "invert": 47.47,
            "height": 1.5,
            "width": 2.5
        },
        "friction": {
            "friction_eq": "MANNING",
            "friction_set": [
                0.01,
                0.015,
                0.02
            ],
            "all_friction": [
                0.01,
                0.02,
                0.015
            ]
        }
    },
    "(S4_W,WEIR)": {
        "name": "S4_W",
        "type": "WEIR",
        "subtype": null,
        "comment": "",
        "dimensions": {
            "weir_elevation": 49.45,
            "weir_breadth": 4.0
        }
    }
}